docs/scripts/generateProptypes.ts

```diff
@@ -67,6 +67,10 @@ async function generateProptypes(project: XTypeScriptProject, sourceFile: string
         'unstableEndFieldRef',
         'series',
         'axis',
+        'bottomAxis',
+        'topAxis',
+        'leftAxis',
+        'rightAxis',
       ];
```

Comment:
I removed them from the resolved types because they were too complex to be readable

I plan to add here links to `ChartsXAxis` props API, but it is blocked by https://github.com/mui/material-ui/pull/41950